themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl

```diff
@@ -111,12 +111,12 @@
   </div>
 
   <!--  product left body: description -->
-  <div class="product-line-grid-right product-line-actions col-md-5 col-xs-12">
+  <div class="product-line-grid-right product-line-actions col-md-5 col-12">
     <div class="row">
-      <div class="col-xs-4 hidden-md-up"></div>
-      <div class="col-md-10 col-xs-6">
+      <div class="col-4 hidden-md-up"></div>
+      <div class="col-md-10 col-6">
         <div class="row">
-          <div class="col-md-6 col-xs-6 qty">
+          <div class="col-6 qty">
```

Comment:
Missing `col-md-6`

Reply:
here is the same, `col-md-6` is useless which is also 50% width
cf the doc here : https://v4-alpha.getbootstrap.com/layout/grid/